var/spack/repos/builtin/packages/libyogrt/package.py

```diff
@@ -52,9 +54,43 @@ def configure_args(self):
 
         sched = self.spec.variants['scheduler'].value
         if sched != "system":
-            args.append('--with-%s=%s' % (sched, self.spec[sched].prefix))
+            if sched == "lsf":
+                # The LSF library depends on a couple of other libraries,
+                # and running the build inside of spack does not find
+                # them, and the user has to add them when they want
+                # to use -lyogrt. If we explicitly tell it what libraries
+                # to use, the user does not need to specify them
+                args.append('--with-lsf')
+                args.append('LIBS=-llsf -lrt -lnsl')
+            else:
+                args.append('--with-%s=%s' % (sched, self.spec[sched].prefix))
 
         if '+static' in self.spec:
             args.append('--enable-static=yes')
 
         return args
+
+    def install(self, spec, prefix):
+        # Run standard Autotools install first, then create a yogrt.conf file
+        # for the selected scheduler
+
+        AutotoolsPackage.install(self, spec, prefix)
+
+        etcpath = os.path.join(prefix, "etc")
+
+        # create subdirectory to hold yogrt.conf file
+        if not os.path.isdir(etcpath):
+            mode = 0o755
+            os.mkdir(etcpath, mode)
+
+        # if no scheduler is specified, create yogrt conf file
+        # with backend=none
+        sched = self.spec.variants['scheduler'].value
+        if sched == "system":
+            sched = "none"
+
+        # create conf file to inform libyogrt about job scheduler
+        with open(os.path.join(etcpath, "yogrt.conf"), "w+") as f:
+            f.write("backend=%s\n" % sched)
+
+        return
```

Comment:
No need to return explicitly at the end of a function
```suggestion
```

Reply:
Agreed